.github/workflows/build-stdpar.yml

```diff
@@ -0,0 +1,90 @@
+name: Build NVHPC stdpar
+
+on:
+  workflow_call:
+    inputs:
+      enable_slack_alerts:
+        description: "If true, a message will be posted to the CCCL GHA CI Alert channel if the workflow fails."
+        required: false
+        default: false
+        type: boolean
+
+jobs:
+  build-stdpar:
+    name: Build NVHPC stdpar
+    runs-on: linux-amd64-gpu-rtx2080-latest-1
+    permissions:
+      id-token: write
+      contents: read
+    steps:
+      - name: Checkout repo
+        uses: actions/checkout@v4
+        with:
+          fetch-depth: 0
+          persist-credentials: false
+      - name: Add NVCC problem matcher
+        run: echo "::add-matcher::$(pwd)/.github/problem-matchers/problem-matcher.json"
+      - uses: aws-actions/configure-aws-credentials@v4
+        with:
+          role-to-assume: arn:aws:iam::279114543810:role/gha-oidc-NVIDIA
+          aws-region: us-east-2
+          role-duration-seconds: 43200 # 12h
+      - name: Run command
+        run: |
+          cat <<"EOF" > "$RUNNER_TEMP/ci.sh"
+          #! /usr/bin/env bash
+          set -eo pipefail
+
+          ~/cccl/ci/stdpar/build_stdpar.sh;
+          sccache --show-adv-stats
+          EOF
+
+          chmod +x "$RUNNER_TEMP/ci.sh"
+
+          mkdir -p .aws
+
+          cat <<EOF > .aws/config
+          [default]
+          bucket=rapids-sccache-devs
+          region=us-east-2
+          EOF
+
+          cat <<EOF > .aws/credentials
+          [default]
+          aws_access_key_id=$AWS_ACCESS_KEY_ID
+          aws_session_token=$AWS_SESSION_TOKEN
+          aws_secret_access_key=$AWS_SECRET_ACCESS_KEY
+          EOF
+
+          chmod 0600 .aws/credentials
+          chmod 0664 .aws/config
+
+          .devcontainer/launch.sh \
+            --docker \
+            --cuda 12.5 \
+            --host nvhpc \
```

Comment:
```suggestion
            --host nvhpc24.7 \
```

Reply:
Shouldnt we be on 25.1?

Reply:
We only have [devcontiners](https://github.com/rapidsai/devcontainers/blob/branch-25.04/matrix.yml#L25-L26) for 24.7 and 24.9 at the moment,. The other nvhpc tests I saw are using 24.7 so that's why I picked it, but my main point was that a version needs to be specified for this to work.

Reply:
It would be helpful if the devcontainers were kept up to date with the latest NVHPC releases, and if there were a way to say "nvhpc-latest" without having to specify an exact version.  NVHPC releases come out on average every two months.

Reply:
Our devcontainer update process is unfortunately a rather manual process, so we can't just automate this. But I'll look into building some new containers with the latest version today.